java/data-management/arrays/copying-an-array-with-system-arraycopy.md

```diff
@@ -2,54 +2,118 @@
 author: adamMontgomerie
 
 levels:
-
   - basic
-
   - advanced
-
   - medium
 
 type: normal
 
 category: tip
 
 aspects:
-
   - introduction
-
   - obscura
 
 tags:
-
   - loops
-
   - arrays
 
 links:
-
   - '[www.javapractices.com](http://www.javapractices.com/topic/TopicAction.do?Id=3){website}'
 
 ---
 
-# Copying an array with `System.arraycopy()`
+# Copying an Array
 
 ---
 ## Content
 
-`System.arraycopy` can be used to copy the contents of an array to a new location in a more concise way than using a loop. For example, to copy content from `src` to `dest.`
+To copy the contents of an array, you can use the  `Arrays.copyOf()` method. It can be found in the `java.util.Arrays` class, and the syntax is:
```

Comment:
```suggestion
To copy the contents of an array, you can use the `Arrays.copyOf()` method. It can be found in the `java.util.Arrays` class, and the syntax is:
```